### PR TITLE
[Term Entry] PHP String function: rtrim()

### PR DESCRIPTION
**Pull request by sarvesh-7**
### Description

- Added String function rtrim() term for PHP
- Closes issue #1898 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+- The `string` parameter is the string from which we need to remove the characters.
+- The `charlist` parameter specifies which characters to remove from the string. If omitted then following characters are removed.
+
+ - **"\O"** - NULL
+ - **"\t"** - tab
+ - **"\n"** - new line
+ - **"xOB"** - vertical tab
+ - **"\r"** - carriage return
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/rtrim/rtrim.md:30]:
```suggestion
 - `\r` - carriage return
```

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ - **" "** - whitespace
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/rtrim/rtrim.md:31]:
```suggestion
 - `" "` - space
```

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/rtrim/rtrim.md:33]: We generally only have one codebyte example per entry. Remove one of these, or make one a regular `## Example` with sample output.

File: content/html/concepts/attributes/terms/id/id.md
Comment (SSwiniarski) [inline content/html/concepts/attributes/terms/id/id.md:1]: Undo this change.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@sarvesh-7 I did a first review and left a few comments.

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - 'Strings'
+    - 'Functions'
+CatelogContent:
+    - 'learn-php'
+    - 'paths/computer-science'
+---
+
+The **`rtrim()`** method removes whitespace or other predefined characters from the right side of a string.
+
+## Syntax
```
[the remark this answers is not in the document]
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md:18]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+```psuedo
+rtrim(string,charlist)
+```
+
+- The `string` parameter is the string from which we need to remove the characters.
+- The `charlist` parameter specifies which characters to remove from the string. If omitted then following characters are removed.
+
+ - **"\O"** - NULL
```
[the remark this answers is not in the document]
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+ - **"\t"** - tab
```
[the remark this answers is not in the document]
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+ - **"\n"** - new line
```
[the remark this answers is not in the document]
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md (hunk 2 above)
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md (hunk 3 above)
Comment (sarvesh-7) [reply content/php/concepts/string-functions/terms/rtrim/rtrim.md]: made one a regular ## Example with sample output

File: content/html/concepts/attributes/terms/id/id.md
Comment (sarvesh-7) [reply content/html/concepts/attributes/terms/id/id.md:1]: done

File: content/php/concepts/string-functions/terms/rtrim/rtrim.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
+---
+Title: 'rtrim()'
+Description: 'removes whitespace or other predefined characters from the right side of a string.'
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/rtrim/rtrim.md:3]:
```suggestion
Description: 'Removes whitespace or other predefined characters from the right side of a string.'
```

**Review by SSwiniarski — APPROVED** [review]
@sarvesh-7 aside from one comment we're good for a second review.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@sarvesh-7 live link: https://www.codecademy.com/resources/docs/php/string-functions/rtrim

Screenshot 2023-05-01 at 1 39 02 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> 
> 
> Make sure to set your visibility to “everyone”.
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin

> 
> If the steps above do not work, it may be due to an email or local configuration issue.
> 
> In addition, we may reach out in the future if any adjustments to these entries are required.
>

cc: @SSwiniarski @KTom101
